public/profile.css

```
@@ -0,0 +1,50 @@
+body{
```

ここスペース欲しいです！！
他にも見受けられるので確認お願いします！
```suggestion
body {
```